test-files/cases/roundabouts/unifying/roundabout-003/Expected.osm

```diff
@@ -179,6 +179,7 @@
     </node>
     <node visible="true" id="135" timestamp="2017-06-07T15:16:56Z" version="1" changeset="117" lat="18.5490360000000045" lon="-72.3467378757499944">
```

Comment:
ID only change. Also addition of `hoot:hash` for some nodes (other tests got that too). I'm ok with that b/c that's a legitimate hoot metadata tag and we have those all over test output since they help in debugging.